app/javascript/packages/components/field-set.tsx

```diff
@@ -0,0 +1,21 @@
+import { FieldsetHTMLAttributes, ReactNode } from 'react';
+
+export interface FieldSetProps extends FieldsetHTMLAttributes<HTMLFieldSetElement> {
```

Comment:
Really nice work with these reusable components 👍 

Interesting on the difference in capitalization between "Fieldset" and "FieldSet", even between React and TypeScript's built-in DOM typings. I'm not sure which is "correct", though I think I'd agree with you that its spelled-out ["field set"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) as two words would make sense capitalized as you've proposed.